dotcom-rendering/src/web/components/TopicFilterBank.tsx

```diff
@@ -62,19 +62,48 @@ const getKeyEventLink = (filterKeyEvents: boolean, id: Props['id']) => {
 	return `?${urlParams.toString()}#${id}`;
 };
 
-const isEqual = (selectedTopic: Topic, availableTopic: Topic) =>
-	availableTopic.type === selectedTopic.type &&
-	availableTopic.value === selectedTopic.value;
+const isEqual = (selectedTopic: Topic | undefined, availableTopic: Topic) => {
+	if (!selectedTopic) return false;
+	return (
+		availableTopic.type === selectedTopic.type &&
+		availableTopic.value === selectedTopic.value
+	);
+};
 
-const getTopFiveTopics = (availableTopics: Topic[]) =>
-	availableTopics
+// get top 5 only if they occur in more than 2 blocks
+const getTopFiveTopics = (availableTopics: Topic[]) => {
+	return availableTopics
 		.slice(0, 5)
 		.filter((topic) => topic.count !== undefined && topic.count > 2);
+};
 
 export const hasRelevantTopics = (availableTopics?: Topic[]) => {
 	return !!(availableTopics && getTopFiveTopics(availableTopics).length);
 };
 
+export const getTopFiveIncludingSelected = (
+	selectedTopic: Topic | undefined,
+	availableTopics: Topic[],
+) => {
+	const topFiveTopics = getTopFiveTopics(availableTopics);
+
+	const selectedIndex = availableTopics.findIndex((availableTopic) =>
+		isEqual(selectedTopic, availableTopic),
+	);
```

Comment:
You can skip checking the array completely if you know that `selectedTopic` is `undefined`

```ts
const selectedIndex = selectedTopic ?
	availableTopics.findIndex((availableTopic) =>
		isEqual(selectedTopic, availableTopic),
	) : -1;
```

Reply:
Great suggestion, made the change 👍